resources/views/userguide.blade.php

```diff
@@ -0,0 +1,216 @@
+							<ul>
+								<li>
+									<p class="card-text" style="color:black;">From the top part of the menu, you can pick a color for each <b>Step</b> to visualize your workflow.</p>
+								</li>
+								<li>
+									<p class="card-text" style="color:black;">To define the <b>Step</b> you can add a title and a description. Click on   <img  src="/images/pencil.svg" alt="Edit"  style="width:16px;height:16px;"> to edit and click on <img src="/images/check.svg" alt="Edit" class="buttonIcon right" style="width:16px;height:16px;"> to save.</p>
+								</li>
+								<li>
+									<p class="card-text" style="color:black;">It is extremely important to define the type of your <b>Step</b> (whether it is an <b>Input Step</b> or a <b>Result Step</b>).</p>
+								</li>
+							</ul>
+							<img src="{{ URL::to('/images/instructions_5.JPG') }}">
+						</li>
+						</div>
+						
+						<br>
+
+						<div id="section8">
+						<li>
+							<h6><b>Variables Menu</b></h6>
+							<p class="card-text" style="color:black;">You can pick the variables to be used in this <b>Step</b> using the <b>Variables Menu</b>.</p>
+							<ul>
+								<li>
+									<p class="card-text" style="color:black;">Choose the variables you want to add using the dropdown menu. </p>
+
+									<img src="{{ URL::to('/images/instructions_6a.JPG') }}">
+								</li>
+								<li>
+									<p class="card-text" style="color:black;">You can always discard the variable using the red cross next to it.</p>
+
+									<img src="{{ URL::to('/images/instructions_6b.JPG') }}">
+								</li>
+								<li>
+									<p class="card-text" style="color:black;">Once you pick your variables, you can edit their title and description way by clicking on them. Click on   <img  src="/images/pencil.svg" alt="Edit"  style="width:16px;height:16px;"> to edit and click on <img src="/images/check.svg" alt="Edit" class="buttonIcon right" style="width:16px;height:16px;"> to save. </p>
+
+									<img src="{{ URL::to('/images/instructions_6c.JPG') }}">
+								</li>
+							</ul>
+						</li>
+						</div>
+
+						<br>
+
+						<div id="section9">
+						<li>
+							<h6><b>Model Calculation Menu</b></h6>
+							<p class="card-text" style="color:black;">When you are done with your variables, you can match your variables and Evidencio variables for result calculation of that Step using the <b>Model calculation</b> menu.</p>
+							<ul>
+								<li>
+									<p class="card-text" style="color:black;">Choose the model you want to use using the dropdown menu. </p>
+
+									<img src="{{ URL::to('/images/instructions_7a.JPG') }}">
+								</li>
+								<li>
+									<p class="card-text" style="color:black;">You can always discard model using the red cross next to it.</p>
+
+									<img src="{{ URL::to('/images/instructions_7b.JPG') }}">
+								</li>
+								<li>
+									<p class="card-text" style="color:black;">Match your new variables with your Evidencio Model variables.</p>
```

Comment:
This matching is done automatically as much as possible, your current pictures are slightly outdated. Not sure if this is a problem.